src/views/ViewUtils.ts

```diff
@@ -184,23 +174,6 @@ export class ViewUtils {
     });
   }
 
-  static findVisynViews(idType?: IDType): Promise<VisynViewPluginDesc[]> {
```

Comment:
@puehringer  How do we achieve the same thing again, since this got removed, and i did not see anything similar in visyn_core.
`PluginRegistry.listPlugins()` and then filter?

Reply:
Yep, simply move this function to your codebase (or use PluginRegistry.listPlugins + filter).